File: year2/systems-design-implementation/Lab4/Lab4Sharp/Domain/Participant.cs
using Lab2Sharp.Utils;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lab2Sharp.Domain
{
    public class Participant : Entity<int>
    {
        public Participant(string nume, int varsta)
        {
            this.nume = nume;
            this.varsta = varsta;
        }
        public Participant(int id, string nume, int varsta)
        {
            base.id = id;
            this.nume = nume;
            this.varsta = varsta;
        }

        public String nume { get; set; }
        public int varsta { get; set; }

        public override string ToString()
        {
            return base.ToString() + " " + nume + ", " + varsta;
        }
    }
}

File: year2/systems-design-implementation/Lab2/Lab2/src/main/java/domain/Participant.java
package domain;

import utils.Entity;

public class Participant extends Entity<Integer> {

    private String nume;
    private Integer varsta;

    public Participant(Integer id, String nume, Integer varsta) {
        super(id);
        this.nume = nume;
        this.varsta = varsta;
    }

    public String getNume() {
        return nume;
    }

    public void setNume(String nume) {
        this.nume = nume;
    }

    public Integer getVarsta() {
        return varsta;
    }

    public void setVarsta(Integer varsta) {
        this.varsta = varsta;
    }

    @Override
    public String toString() {
        return super.toString() + " Participant{" +
                "nume='" + nume + '\'' +
                ", varsta=" + varsta +
                '}';
    }

}
